resources/views/components/expander.blade.php

```diff
@@ -1,7 +1,7 @@
 <div class="expander" x-data="{expanded: false}">
     <x-header class="expander__summary" :level="$level">
-        <button x-bind:aria-expanded="expanded.toString()" x-on:click="expanded = !expanded">
-            {{ $summary }} <x-heroicon-s-plus x-show="! expanded" aria-hidden="true" class="icon" /><x-heroicon-s-minus x-show="expanded" aria-hidden="true" class="icon" />
+        <button type="button" x-bind:aria-expanded="expanded.toString()" x-on:click="expanded = !expanded">
+            {{ $summary }} <span class="badge">{{ __('Content added') }}</span> <x-heroicon-s-plus x-show="! expanded" aria-hidden="true" class="icon" /><x-heroicon-s-minus x-show="expanded" aria-hidden="true" class="icon" />
```

Comment:
I believe initially you were going to have the "badge" be an aria-describedBy for the button instead of being part of the button. Was that idea changed, or did you mean to implement this like that?

Reply:
Oops, this is a case of me having edited the wrong file. Fixing now.